package.json

```diff
@@ -81,6 +81,7 @@
     "chai-as-promised": "^7.1.1",
     "chai-bignumber": "^2.0.2",
     "chai-string": "^1.4.0",
+    "ethereumjs-util": "^5.2.0",
```

Comment:
Do you need to add this?

Reply:
It is required for the MerkleTree utils used in the battleship test.